source/releasenotes/2024-10-14-new-relic-agent.md

```diff
@@ -0,0 +1,14 @@
+---
+title: New Relic PHP agent v11.2.0.15 now available
+published_date: "2024-10-14"
+categories: [tools-apis]
+---
+
+The New Relic PHP agent has been updated to the latest version, v11.2.0.15, for all eligible versions of PHP. Support for PHP 7.0 and 7.1 was removed in version 11.0, so the New Relic PHP agent for those versions remains at v10.22.0.12.
+
+This update brings several enhancements and fixes compared to the previous version, v10.22.0.12, including support for Drupal 11. Note, New Relic agent 11.2 introduces information collection for Composer packages, but this is not yet enabled on Pantheon.
```

Comment:
Since the previous version is mentioned in the previous sentence, I don't think we need to repeat it here.

```suggestion
This update brings several enhancements and fixes compared to the previous version including support for Drupal 11. Note, New Relic agent 11.2 introduces information collection for Composer packages, but this is not yet enabled on Pantheon.
```